src/api/api.ts

```diff
@@ -0,0 +1,61 @@
+import { openmrsFetch } from '@openmrs/esm-api';
+import { careSetting } from '../constants';
+import { Order, OrderPost } from '../types/order';
+
+const DURATION_UNITS_CONCEPT = '1732AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
```

Comment:
Personally I prefer `durationUnitsConcept` here. The all caps etc. style is something that was used exclusively in C to not mix macros with other parts.

Reply:
(one use of a macro was a compile-time constant, i.e., just a constant. that concept does not exist in JS - the `const` is not a compile-time constant, but rather a variable that cannot be reassigned)